pygmt/src/meca.py

```diff
@@ -221,6 +221,50 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
     r"""
     Plot focal mechanisms.
 
+    The following focal mechanism conventions are supported:
+
+    .. list-table:: Supported focal mechanism conventions.
+       :widths: 15 15 40 30
+       :header-rows: 1
+
+       * - Convention
+         - Description
+         - Focal parameters
+         - Remark
+       * - ``"aki"``
+         - Aki and Richard
+         - *strike*, *dip*, *rake*, *magnitude*
+         - angles in degrees
+       * - ``"gcmt"``
+         - global centroid moment tensor
+         - | *strike1*, *dip1*, *rake1*,
+           | *strike2*, *dip2*, *rake2*,
+           | *mantissa*, *exponent*
+         - | angles in degrees;
+           | *mantissa*, *exponent*are for
+           | seismic moment in dyne-cm
```

Comment:
$`moment=mantissa * 10 ^ {exponent}`$. 

Maybe we should rewrite this note to:
```suggestion
           | seismic moment is :math:`mantissa * 10 ^ {exponent}`
           | in dyne-cm
```